deploy/crds/openebs.io_blockdevices_crd.yaml

```diff
@@ -13,7 +13,28 @@ spec:
     singular: blockdevice
   scope: Namespaced
   versions:
-  - name: v1alpha1
+  - additionalPrinterColumns:
+    - jsonPath: .spec.nodeAttributes.nodeName
+      name: NodeName
+      type: string
+    - jsonPath: .spec.capacity.storage
+      name: Size
+      type: string
+    - jsonPath: .status.claimState
+      name: ClaimState
+      type: string
+    - jsonPath: .status.state
+      name: Status
+      type: string
+    - jsonPath: .spec.path
+      name: Path
+      priority: 7
+      type: string
+    - jsonPath: .spec.filesystem.fsType
+      name: FSType
+      priority: 8
+      type: string
```

Comment:
This needs to be reordered as specified here. https://github.com/openebs/node-disk-manager/blob/7e50f5b2e80fbf7efcff86b20f91b3e12686e687/pkg/setup/build_crd.go#L36.

Also, why is the priority set at 7 and 8. Just 1 would sufice.